mu/modes/circuitpython.py

```diff
@@ -72,6 +72,8 @@ class CircuitPythonMode(MicroPythonMode):
         (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
         (0x303A, 0x8002, None, "Unexpected Maker TinyS2"),
         (0x054C, 0x0BC2, None, "Spresense"),
+        (0x303A, 0x80AA, None, "Franzininho WiFi Wroom"),
+        (0x303A, 0x80AD, None, "Franzininho WiFi Wrover"),        
```

Comment:
Thanks! I can confirm the PIDs from the Espressif repo:
- https://github.com/espressif/usb-pids/blob/d40fe6f7369261db0f5394c5c4bf24311bc6184d/allocated-pids.txt#L178
- https://github.com/espressif/usb-pids/blob/d40fe6f7369261db0f5394c5c4bf24311bc6184d/allocated-pids.txt#L181